NewInternalsSpecification.html

```diff
@@ -0,0 +1,271 @@
+          1. If _O_ does not have a [[Target]] internal slot return.
+          1. Let _type_ be Get(_changeRecord_, `"type"`).
+          1. ReturnIfAbrupt(_type_).
+          1. If Type(_type_) is not string, throw a *TypeError* exception.
+          1. Let _target_ be the value of _O_'s [[Target]] internal slot.
+          1. Let _newRecord_ be ObjectCreate(%ObjectPrototype%).
+          1. Let _desc_ be the PropertyDescriptor{[[Value]]: _target_, [[Writable]]: *false*, [[Enumerable]]: *true*, [[Configurable]]: *false*}.
+          1. Let _success_ be the result of calling the [[DefineOwnProperty]] internal method of _newRecord_ passing `"object"` and _desc_ as arguments.
+          1. Assert: _success_ is *true*.
+          1. Let _names_ be the result of calling the [[Enumerate]] internal method of _changeRecord_ with no arguments.
+          1. Repeat for each element _N_ of _names_ in List order,
+            1. If _N_ is not `"object"`, then
+              1. Let _value_ be Get(_changeRecord_, _N_).
+              1. ReturnIfAbrupt(_value_).
+              1. Let _desc_ be the PropertyDescriptor{[[Value]]: _value_, [[Writable]]: *false*, [[Enumerable]]: *true*, [[Configurable]]: *false*}.
+              1. Let _success_ be the result of calling the [[DefineOwnProperty]] internal method of _newRecord_ passing _N_ and _desc_ as arguments.
+              1. Assert: _success_ is *true*.
+          1. Set the value of the [[Extensible]] internal slot of _newRecord_ to *false*.
+          1. Call EnqueueChangeRecord(_target_, _newRecord_).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="notifierprototype_.performchange">
+        <h1>%NotifierPrototype%.performChange(changeType, changeFn)</h1>
+        <emu-alg>
+          1. Let _O_ be the *this* value.
+          1. If Type(_O_) is not Object, throw a *TypeError* exception.
+          1. If _O_ does not have a [[Target]] internal slot return.
+          1. 1. Let _target_ be the value of _O_'s [[Target]] internal slot.
+          1. If Type(_changeType_) is not string, throw a *TypeError* exception.
+          1. If IsCallable(_changeFn_) is *false*, throw a *TypeError* exception.
+          1. Call BeginChange(_target_, _changeType_).
+          1. Let _changeRecord_ be the result of calling the [[Call]] internal method of _changeFn_, with *undefined* as _thisArgument_ and an empty List as _argumentsList_.
+          1. Call EndChange(_target_, _changeType_).
+          1. ReturnIfAbrupt(_changeRecord_).
+          1. Let _changeObservers_ be the value of _O_'s [[ChangeObservers]] internal slot.
+          1. If _changeObservers_ is empty, return.
+          1. Let _target_ be the value of _O_'s [[Target]] internal slot.
+          1. Let _newRecord_ be ObjectCreate(%ObjectPrototype%).
+          1. Let _desc_ be the PropertyDescriptor{[[Value]]: _target_, [[Writable]]: *false*, [[Enumerable]]: *true*, [[Configurable]]: *false*}.
```

Comment:
Stylesheet needs to bold emu-vals. They are outputting correctly but not visually distinct.